lms/djangoapps/courseware/views/index.py

```diff
@@ -183,7 +183,7 @@ def _verify_position(self):
         """
         if self.position is not None:
             try:
-                int(self.position)
+                self.position = max(int(self.position), 1) # test change
```

Comment:
It's also good to get into the habit of linking to the ticket you're working on, just for convenience.